book/src/asm/functions.md

```diff
@@ -1,18 +1,16 @@
 # Functions
 
-Machine functions are the entry points to a machine. They can be called from another machine or from the outside.
+Functions are the entry points to a virtual machine. They can be called from another machine or from the outside.
```

Comment:
What does outside mean here actually, besides another machine?

Reply:
From a snark verifier basically, for the entry point.